vars/buildStripesPlatform.groovy

```diff
@@ -9,11 +9,11 @@ def call(String okapiUrl, String tenant, String branch='') {
 
   def foliociLib = new org.folio.foliociCommands()
 
-  if (!(branch =~ /^(r|R)\d{1}-\d{4}(-(rc|RC|hotfix-\d{1}))?$/)) {
+  if (!(branch =~ /^[rR]\d-\d{4}(-([rR][cC]|hotfix-\d))?/)) {
```

Comment:
We should be using branches named like `2023-R2...` instead of `R2...` that do not sort correctly. Is this an opportunity to correct that gaffe or has that ship already sailed?